src/js/components/Box/__tests__/Box-layout-test.tsx

```diff
@@ -31,6 +36,56 @@ describe('Box', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('responsive container medium', () => {
+    jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect').mockReturnValue({
+      width: 1280,
+      height: 0,
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
+      x: 0,
+      y: 0,
+      toJSON: () => {},
+    });
+    const { container } = render(
+      <Grommet>
+        <Box responsive="container" width="1280px">
+          <ResponsiveContext.Consumer>
+            {(size2) => size2}
```

Comment:
How come "size2" instead of just "size"?

Reply:
Not needed at this point but I wasn't convinced both size vars weren't in the same scope. Long story, jest dom doesn't really render content fully. As part of the test `render` the getBoundingClientRect will always return 0's and no observers will get triggered (hence the spy.) So while checking this I tried using different vars for the return value to make sure one didn't overwrite the other. The seem to end up as differently scoped vars in the JSX :)